test-app.gradle

```diff
@@ -17,8 +17,12 @@ private static void apply(Settings settings) {
     }
 
     settings.include(":app")
+    settings.include(":support")
```

Comment:
Is there no way to leverage autolinking to pull this in? How will the consumer use this? Can we implement an example in the Example app?

Reply:
The `support` module will be automatically linked for consumers, so no additional work will be required. To start receiving events, the consumer's `ReactPackage` has to implement the provided interface.

> Can we implement an example in the Example app?

Of course, but we will need to add native modules to iOS/Android examples to implement the interface. I don't remember why we've decided to keep the example as "JS" only.